src/components/Layout.tsx

```diff
@@ -0,0 +1,171 @@
+import { useAuthProvider } from 'Auth/MAPI/MapiAuthProvider';
```

Comment:
Refactored this into a functional component (and TypeScript), to be able to use hooks